core/src/script/CGXP/widgets/tree/LayerTree.js

```diff
@@ -404,6 +411,17 @@ cgxp.tree.LayerTree = Ext.extend(Ext.tree.TreePanel, {
 
                 var timeWidget;
                 if (item.time) {
+                    if (this.initialState && this.initialState['time_' + item.name]) {
+                      var times = this.initialState['time_' + item.name].split('/');
+                      if (times.length === 2) {
```

Comment:
The WMS time support many possibilities:
http://mapserver.org/ogc/wms_time.html#supported-time-requests
But with GeoMapFish we support only single and range :-)
https://github.com/camptocamp/c2cgeoportal/blob/master/doc/integrator/wmstime.rst#admin-interface